docs: Russian translation moved to VuePress

lex111 opened the pull request:
cc @Alex-Sokolov

lex111 committed:
Migrate Russian docs to VuePress

lex111 committed:
Russian docs: video link for common tips

sync with https://github.com/vuejs/vue-test-utils/commit/c0b2101f75b80e1dd5691656d0929085d5adb886#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 committed:
Russian docs: overwrite arrays in setData

sync with https://github.com/vuejs/vue-test-utils/commit/032a7a4e7910ff7c8f8c659a90ff921cfd478d0f#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 committed:
Russian docs:  remove phantomjs limitation

sync with https://github.com/vuejs/vue-test-utils/commit/e9f3305efae71b3df874a7debedc4963161ed922#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 committed:
Russian translation: actualize docs

lex111 committed:
Merge remote-tracking branch 'origin' into vuepress-ru

lex111 committed:
Russian translation: silence warnings when updating prop

Sync with https://github.com/vuejs/vue-test-utils/commit/7fa2fb390b7b4c5a4105315876a1f72aaf1684bc#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 committed:
Russian translation: add max-len rule

Sync with https://github.com/vuejs/vue-test-utils/commit/2872a201e7ba3dced329177b5a95958de5440540#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 commented:
Updated with the latest changes in the docs.

I look forward to @Alex-Sokolov :stuck_out_tongue_winking_eye:

lex111 committed:
Fix build

lex111 committed:
Fix build

lex111 committed:
Fix menu items

lex111 committed:
Russian translation: fix headings

lex111 committed:
Russian translation: fix all issues

lex111 committed:
Russian translation: more fixes

lex111 committed:
Russian translation: fix headings of guides

lex111 committed:
Russian translation: fixes for wrapper-array

lex111 commented:
Ready for review

Alex-Sokolov commented:
@lex111 great work! Thanks :)

38elements commented on docs/ru/api/createLocalVue.md:
````diff
@@ -0,0 +1,27 @@
+
+```js
+import { createLocalVue, shallow } from '@vue/test-utils'
+import Foo from './Foo.vue'
````
shallow -> shallowMount

lex111 replied:
@38elements  thank you! I have not everywhere before made a replacement, now thanks to you, I replaced everywhere with `shallow` on `shallowMount`

lex111 committed:
Russian translation: replace shallow with shallowMount

lex111 commented:
@eddyerburgh we are a little delayed :disappointed: , but now the documentation in Russian is relevant, look, please.

38elements commented on docs/ru/api/config.md:
```diff
@@ -0,0 +1,111 @@
+По аналогии с `stubs`, значения, переданные в `config.mocks` используются по умолчанию. Любые значения, переданные настройкам монтирования объекта `mocks`, будут иметь приоритет выше, по сравнению с объявленными в `config.mocks`.
+
+Example:
+
```
Example -> Пример

lex111 committed:
Russian translation: minor fixes

38elements commented on docs/ru/api/config.md:
```diff
@@ -0,0 +1,111 @@
+
+### `mocks`
+
+- type: `Object`
```
type -> Тип

38elements commented on docs/ru/api/config.md:
```diff
@@ -0,0 +1,111 @@
+### `mocks`
+
+- type: `Object`
+- default: `{}`
```
default -> по умолчанию

lex111 committed:
Russian translation: fixes

lex111 commented:
@38elements thanks for the review.

38elements commented on docs/ru/api/config.md:
```diff
@@ -0,0 +1,111 @@
+Заглушки, используемые в компонентах. Они перезаписываются значениями `stubs` переданными в настройках монтирования.
+
+При передаче `stubs` в качестве массива в настройках монтирования, `config.stubs` будет преобразована в массив, и будут создаваться компоненты заглушки с базовым компонентом, который возвращает div.
+
```
div -> <${component name}-stub>
https://vue-test-utils.vuejs.org/api/config.html#stubs

lex111 committed:
Russian translation: fixes

38elements commented on docs/ru/api/renderToString.md:
```diff
@@ -0,0 +1,101 @@
+
+См. [опции](./options.md)
+
+- **Usage:**
```
Usage -> Использование

lex111 committed:
Russian translation: fixes

38elements commented:
#368 is not included.
IMHO, I think this needs to check well.

lex111 committed:
Merge branch 'dev' of https://github.com/vuejs/vue-test-utils into vuepress-ru

lex111 committed:
Russian translation: tweaks

Sync with https://github.com/vuejs/vue-test-utils/commit/08b36cefb4d419735180f681faf5f93188e86194

lex111 committed:
Russian translation: silentWarnings to silent

Sync with https://github.com/vuejs/vue-test-utils/commit/7302e055422d3043b322049b9058b34567154152#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 committed:
Russian translation: improve document for setChecked(), setSelected() and setValue()

Sync with https://github.com/vuejs/vue-test-utils/commit/c78ed4ed2fc66f79b50d9858eafb525666e06cdc#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 committed:
Russian translation: complete update in accordance with the original version

lex111 committed:
Russian translation: fix lint error

lex111 commented:
Sorry for the large number of inconsistencies, now I carefully checked everything and corrected the errors. I also updated the Russian translation in accordance with the new changes (_silentWarnings to silent_ and others)

Special thanks to @38elements for reporting discrepancies!

@Alex-Sokolov can you please check again?

Next time we will update this translation together with community Translation Gang, now I acted alone, so it took a long time :face_with_head_bandage:

38elements commented:
#493  is not included.
If the document has many incorrect points, many problems and pull requests will be generated.
This increases maintenance cost.

lex111 committed:
Russian translation: replace from mapCoverage to collectCoverage

Sync with https://github.com/vuejs/vue-test-utils/pull/493/files

lex111 commented:
@38elements thanks, fixed it.

38elements commented:
I took a look at all the files roughly.
I could not find any other problems.
I hope other people review this pull request.

38elements commented:
I took a look at all the files roughly.
I could not find the other problems.
I hope other people review this pull request.

lex111 committed:
Russian translation: fix anchor

Alex-Sokolov commented on docs/ru/api/renderToString.md:
```diff
@@ -0,0 +1,101 @@
+      - `{Array<Component|Object>|Component} children`
+    - `{Object} slots`
+        - `{Array<Componet|Object>|Component|String} default`
+        - `{Array<Componet|Object>|Component|String} named`
```
Component — here and line above

Alex-Sokolov commented on docs/ru/api/render.md:
```diff
@@ -0,0 +1,103 @@
+      - `{Array<Component|Object>|Component} children`
+    - `{Object} slots`
+        - `{Array<Componet|Object>|Component|String} default`
+        - `{Array<Componet|Object>|Component|String} named`
```
Component — here and line above

Alex-Sokolov commented on docs/ru/api/renderToString.md:
```diff
@@ -0,0 +1,101 @@
+    const renderedString = renderToString(Foo, {
+      slots: {
+        default: [Bar, FooBar],
+        fooBar: FooBar, // Will match <slot name="FooBar" />,
```
Not translated yet ("будет соответствовать" in other places)

Alex-Sokolov commented on docs/ru/api/render.md:
```diff
@@ -0,0 +1,103 @@
+    const wrapper = render(Foo, {
+      slots: {
+        default: [Bar, FooBar],
+        fooBar: FooBar, // Will match <slot name="FooBar" />,
```
Not translated yet ("будет соответствовать" in other places)

Alex-Sokolov commented:
Also deploy on netlify failed because `7:03:08 PM: failed during stage 'deploying site': Failed to execute deploy: Upload cancelled: zh/guides/using-with-vuex.html`. Maybe need to update some link inside.

Alex-Sokolov commented on docs/ru/api/config.md:
```diff
@@ -0,0 +1,111 @@
+- Тип: `Object`
+- По умолчанию: `{}`
+
+Вы можете настроить методы по умолчанию с помощью объекта `config`. Это может быть полезно для плагинов, которые вводят методы в компоненты, такие как [VeeValidate](https://vee-validate.logaretm.com/). Вы можете переопределить методы, установленные в `config`, передав `methods` в настройках монтирования.
```
Also need change link to https://baianat.github.io/vee-validate/

lex111 committed:
Merge remote-tracking branch 'origin' into vuepress-ru

lex111 committed:
Russian translation: docs: update attributes.md

Sync with https://github.com/vuejs/vue-test-utils/commit/3494fc0d358b4df02650b2355c0a31cb9d5a1cb8#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 committed:
Russian translation: feat: element, vnode, vm, and options are read-only

Sync with https://github.com/vuejs/vue-test-utils/commit/b801c256f9a32817c37ee6c5f9e5b5b4b30bb6da#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 committed:
Russian translation: feat: throw error if the read-only property is tried to change

Sync with https://github.com/vuejs/vue-test-utils/commit/fb46268e7c36a45d9d43e8da10f53d5d915576ee#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 committed:
Russian translation: Update README.md

Sync with https://github.com/vuejs/vue-test-utils/commit/4bb7ca68973c01907e09f7729cdf289159d28c95#diff-e3e2a9bfd88566b05001b02a3f51d286

lex111 committed:
Russian translation: fixes after review of @Alex-Sokolov

lex111 commented:
@Alex-Sokolov thanks you for this, all edits have been made + updated the documentation in accordance with the new changes in original docs.

lex111 commented:
@eddyerburgh please pay attention to this PR :open_hands:
